core-web/libs/sdk/angular/next/components/dotcms-block-editor-renderer/blocks/list.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'dotcms-block-editor-renderer-bullet-list',
+    standalone: true,
+    template: `
+        <ul>
+            <ng-content />
+        </ul>
+    `
+})
+export class DotCMSBlockEditorRendererBulletListComponent {}
+
+@Component({
+    selector: 'dotcms-block-editor-renderer-ordered-list',
+    standalone: true,
+    template: `
+        <ol>
+            <ng-content />
+        </ol>
+    `
+})
+export class DotCMSBlockEditorRendererOrderedListComponent {}
```

Comment:
If these components are internal and located inside the `/dotcms-block-editor-renderer/blocks` folder, we can omit the `DotCMSBlockEditorRenderer` prefix. We can use `DotOrderedList` if needed, but if they aren't exposed to end developers, it's fine to use more readable names